godel_plugins/include/godel_plugins/widgets/scan_tool_configuration_window.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SCAN_TOOL_CONFIGURATION_WINDOW_H
+#define SCAN_TOOL_CONFIGURATION_WINDOW_H
+
+#include <godel_plugins/widgets/parameter_window_base.h>
+#include <ui_scan_tool_configuration_window.h>
+#include <godel_msgs/ScanPlanParameters.h>
+
+
+namespace godel_plugins
+{
+
+class ScanPlanConfigWidget : public ParameterWindowBase
+{
+  Q_OBJECT
+public:
+  ScanPlanConfigWidget(godel_msgs::ScanPlanParameters params);
+  godel_msgs::ScanPlanParameters& params() { return params_; }
+  static QStringList quality_metric_list;
```

Comment:
It is my understanding that static class members must be POD as described [here](https://google.github.io/styleguide/cppguide.html#Nonmember,_Static_Member,_and_Global_Functions)